internal/cmd/alpha/alpha.go

```diff
@@ -0,0 +1,39 @@
+package alpha
```

Comment:
I need discuss about moving all code for commands to `alpha` directory. I'm not sure that it's necessary.

Reply:
We decided that directory `alpha` is ok for us.